libs/design-system-core/src/utils/time.ts

```diff
@@ -46,5 +48,37 @@ const formatRelativeTime = (date: string, locale?: string) => {
   }
   return '';
 };
+/**
+ * Returns the date in format: 'DD MMM HH:mm'
+ * If the date is today it returns the hours or minutes passed,
+ * If the date is yesterday then it returns this format: [Yesterday] HH:mm
+ */
+const formatRelativeDateTime = (date: string, locale?: string) => {
+  if (dayjs(date).isValid()) {
+    let time = dayjs(date);
+    if (/^[0-9]*$/.test(date)) {
+      time = date.length > 10 ? dayjs(+date) : dayjs.unix(+date);
+    }
+
+    if (locale) {
+      time = time.locale(locale);
+    }
+
+    const now = dayjs();
+    if (time.isSame(now, 'day')) {
+      // If the date is today
+      return time.fromNow();
+    } else if (time.isSame(now.subtract(1, 'day'), 'day')) {
+      // If the date is yesterday
```

Comment:
Day js has [plugin functions](https://day.js.org/docs/en/plugin/is-yesterday) that can be used for checking if yesterday/today etc. Might help with cleaner code.